docs/extra_tests.md

```diff
@@ -18,9 +18,9 @@ not complete before the timeout.
 ```bash
 ./setup build --test --extra
 ./setup test --extra
-# To exclude the Tor tests (for example if you built Shadow in debug mode)
-./setup test --extra -- --label-exclude tor
+# To exclude the TGen and Tor tests (for example if you built Shadow in debug mode)
+./setup test --extra -- --label-exclude tgen|tor
 ```
 
-If you change the version of tor located at `~/.local/bin/tor`, make sure to
-re-run `./setup build --test`.
+If you change the version of tgen located at `~/.local/bin/tgen` or tor located
+at `~/.local/bin/tor`, make sure to re-run `./setup build --test`.
```

Comment:
The reason for this comment about tor is that we run `tor --list-torrc-options` to check what options are available in `src/test/tor/minimal/CMakeLists.txt`, and if you change the tor version, you should re-run this check. I don't think we need to re-run anything if the tgen version changes?